lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -95,6 +99,10 @@ WebProcessingServiceCatalogFunction.parameterConverters = [
                 return undefined;
             }
 
+            if (input.Identifier === "regionType") {
```

Comment:
It seems a bit strange to check this here.  Can we handle it just by ordering the converters differently?  i.e. put the `regionType` converter first in the list, and then this converter works on any `LiteralData` that isn't handled by a prior converter?  That sort of thing will be necessary if someone wants to extend the set up parameters from outside TerriaJS, because they won't be able to add an actual `Identifier` check to our existing converters easily.